src/components/interpolatedColorLegend.ts

```diff
@@ -208,6 +226,15 @@ export module Components {
         height: 0
       };
 
+      let numSwatches = InterpolatedColorLegend._DEFAULT_NUM_SWATCHES;
+
+      if (this.expands()) {
+        let textHeight = this._measurer.measure().height;
+        let offset = this._isVertical() ? 2 * padding :  4 * padding - text0Width - text1Width;
+        let fullLength = this._isVertical() ? this.height() : this.width();
+        numSwatches = Math.max(Math.floor((fullLength - offset) / textHeight), numSwatches);
```

Comment:
This can potentially cause a crash if `textHeight` evaluates to 0, as demonstrated here: http://jsfiddle.net/nr0mz1bm/ (comment the line back in)

Let's bail out if the font-size is 0, and add a test for that?